systemtests/python-modules/bareos_unittest/json.py

```diff
@@ -127,31 +133,37 @@ def run_restore(
             run_parameter.append("jobid={}".format(jobid))
         if fileset:
             run_parameter.append("fileset={}".format(fileset))
+            run_parameter.append("select")
         if extra:
             run_parameter.append("{}".format(extra))
-        run_parameter += ["select", "all", "done", "yes"]
-        result = director.call("restore {}".format(" ".join(run_parameter)))
+        run_parameter += ["all", "done", "yes"]
+        restore_cmd = "restore {}".format(" ".join(run_parameter))
+        result = director.call(restore_cmd)
         jobId = result["run"]["jobid"]
         if wait:
             self.wait_job(director, jobId)
         return jobId
 
-    def get_backup_jobid(self, director, jobname, level="F", extra=None):
+    def get_backup_jobid(self, director, jobname, level=None, fileset=None, extra=None):
```

Comment:
Some tests do not set `level` but it seems they do require a full.  Could you double check this ?

E.g. `test_list_media` seems to require a full but does not enforce it now.